stubs/PyScreeze/pyscreeze/__init__.pyi

```diff
@@ -11,10 +12,17 @@ _R = TypeVar("_R")
 # But can't import either, because pyscreeze does not declare them as dependencies, stub_uploader won't let it.
 _MatLike: TypeAlias = Incomplete
 
-useOpenCV: Final[bool]
+PILLOW_VERSION: Final[tuple[int, int, int]]
 RUNNING_PYTHON_2: Final = False
-GRAYSCALE_DEFAULT: Final = False
-scrotExists: Final[bool]
+SCROT_EXISTS: Final[bool]
+GNOMESCREENSHOT_EXISTS: Final[bool]
+
+if sys.platform == "linux":
+    RUNNING_X11: Final[bool]
+    RUNNING_WAYLAND: Final[bool]
```

Comment:
`SCROT_EXISTS` and `GNOMESCREENSHOT_EXISTS` are nice to have since you can tell ahead of time using Pillow's own logic whether those functionality are A) available, and B) will be used by PyScreeze. Which is something you may also want to communicate to the user.
Similar for Wayland vs X11, although that one's a lot more simple.

These are things I'm interested in having ahead of time (without copying the logic) for AutoSplit on Linux (the port's waiting for more refactor, but I have a branch)

`PILLOW_VERSION` and `RUNNING_PYTHON_2` though, yeah maybe not. But also, it's there, and adding to the allowlist would have a similar maintenance cost (must update entry if name changes or is removed)